### PR TITLE
CRIMAP-696 Add manage_no_income page

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change
Add new step behind feature flag
Add to income decision tree, but not in flow yet
Add route with alias, controller, form and errors

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-696

## Screenshots of changes (if applicable)
image

image

## How to manually test the feature
run rails db:migrate
Start a new application
Go to /steps/income/how_does_client_manage_with_no_income

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -83,7 +83,6 @@
     t.jsonb "provider_details", default: {}, null: false
     t.uuid "parent_id"
     t.string "means_passport", default: [], array: true
-    t.string "expected_evidence", default: [], array: true
```
Comment (arthurashman) [inline db/schema.rb:86]: I somehow introduced this in my previous PR - do I need to do something more intelligent than just delete it from the schema.rb?

File: app/views/steps/income/manage_without_income/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <div class="govuk-grid-column-two-thirds">
+  <span class="govuk-caption-xl"><%= t('steps.income.caption') %></span>
+    <%= govuk_error_summary(@form_object) %>
+    <%= step_form @form_object do |f| %>
+        <%= f.govuk_radio_buttons_fieldset(:manage_without_income, legend: { tag: 'h1', size: 'xl' }) do %>
+          <% @form_object.choices.each_with_index do |choice, index| %>
+            <% if choice == ManageWithoutIncomeType::OTHER %>
+              <%= f.govuk_radio_button :manage_without_income, choice.value do %>
+                <% f.govuk_text_area :manage_other_details,
+                                     label: { size: 's' } %>
```
Comment (hiboabd) [inline app/views/steps/income/manage_without_income/edit.html.erb:14]: If you remove the size attribute, the label text will display without the bold styling
```suggestion
                <% f.govuk_text_area :manage_other_details  %>
```

**Review by hiboabd — APPROVED** [review]
Looks good, just left one comment

File: app/views/steps/income/manage_without_income/edit.html.erb (hunk 2 above)
Comment (arthurashman) [reply app/views/steps/income/manage_without_income/edit.html.erb:14]: You're a genius. Literally trying to find this solution for my next page (ngl - didn't realise it was wrong on this page until I started the new one)! Thank you!

**Pull request merged**